site/examples/iframe.tsx

```diff
@@ -104,15 +104,17 @@ const MarkButton = ({ format, icon }) => {
   )
 }
 
-const IFrame = ({ children, ...props }) => {
-  const [contentRef, setContentRef] = useState(null)
-  const mountNode =
-    contentRef &&
-    contentRef.contentWindow &&
-    contentRef.contentWindow.document.body
+const IFrame = ({ children, onLoad = null, ...props }) => {
+  const [iframeBody, setIframeBody] = useState(null)
+
+  const handleLoad = e => {
+    onLoad && onLoad(e)
```

Comment:
```suggestion
    onLoad?.(e)
```

Reply:
I've removed that part completely as not needed